pyaedt/modules/AdvancedPostProcessing.py

```diff
@@ -50,6 +51,19 @@
     )
 
 
+def isnotebook():
```

Comment:
Just to follow snake case convention function naming.
```suggestion
def is_notebook():
```